ospo-mindmap/Content/ospomindmap.md

```diff
@@ -9,6 +9,7 @@
 - Community Engagement
 - Developer Education
 - Individual Contributor
+- Open Source Adviser
 
 ## 🚀 Behavior
```

Comment:
When doing this big update, does anyone have maybe a better name for this **"Behavior"** node?

The sub-nodes are more collaboration examples between organizations rather than behavior.

Reply:
Thanks, @anajsana makes sense. I'll mark this as resolved.